qreu/sendcontext.py

```diff
@@ -61,12 +61,12 @@ def __init__(
             ssl_keyfile=None, ssl_certfile=None, tls=False
     ):
         super(SMTPSender, self).__init__(
-            _host=host,
-            _port=port,
-            _user=user,
-            _passwd=passwd,
-            _ssl_keyfile=ssl_keyfile,
-            _ssl_certfile=ssl_certfile,
+            _host=str(host),
+            _port=int(port),
+            _user=str(user),
+            _passwd=str(passwd),
+            _ssl_keyfile=str(ssl_keyfile),
+            _ssl_certfile=str(ssl_certfile),
```

Comment:
I don't like the solution to cast everything to `str` because this are different results for py2.7 and py3